optuna/pruners/hyperband.py

```diff
@@ -70,20 +78,42 @@ class HyperbandPruner(BasePruner):
     """
 
     def __init__(
-        self, min_resource=1, reduction_factor=3, n_brackets=4, min_early_stopping_rate_low=0
-    ):
-        # type: (int, int, int, int) -> None
-
+        self,
+        max_resource: int = 100,
+        min_resource: int = 1,
```

Comment:
I'm wondering if it isn't more pedagogic to place "min" before "max" in the signature.